utils/generate_backend_completeness.py

```diff
@@ -38,6 +38,8 @@ class Backend(NamedTuple):
 
 EXCLUDE_CLASSES = {"BaseFrame", "Then", "When"}
 
+DIRECTLY_IMPLEMENTED_CLASS_METHODS = {"DataFrame": ["pipe"]}
```

Comment:
This should hold also for Series and Expr if possible 😇

Reply:
Got it, I just adapted it.